src/app/App.tsx

```diff
@@ -9,6 +10,15 @@ function App(): JSX.Element {
         <Route path="/home">
           <Home />
         </Route>
+        <Route path="/search">
+          <Home />
+        </Route>
+        <Route path="/doctor/:id">
+          <DoctorProfile />
+        </Route>
+        <Route path="/">
+          <Home />
+        </Route>
```

Comment:
The home Page should only be used in one page. So either remove `<Route path="/" />` or `<Route path="/home" />`